OMCompiler/Compiler/Template/SimCodeTV.mo

```diff
@@ -303,8 +305,8 @@ package SimCodeVar
   // for setting Variability attribute for FMI-2.0 export
   uniontype Variability
     record CONSTANT end CONSTANT;
-    record FIXED end FIXED;
-    record TUNABLE end TUNABLE;
+    record FIXED "parameters" end FIXED;
+    record TUNABLE "parameters" end TUNABLE;
```

Comment:
The "parameters" hint is probably not useful. You can add a proper description of the different attributes or skip it.